### PR TITLE
Support not creating change with psbt api

### PR DESCRIPTION
**Pull request by Kukks**
This allows you to explicitly specify to donate leftover change to miners.

### CODE REVIEW

File: NBXplorer/Controllers/MainController.PSBT.cs [hunk 1]
```diff
@@ -323,7 +323,15 @@ public partial class MainController
...
 					txBuilder.SendAllRemainingToChange();
 				psbt = txBuilder.BuildPSBT(false);
-				hasChange = psbt.Outputs.Any(o => o.ScriptPubKey == change.ScriptPubKey);
+				hasChange =  psbt.Outputs.Any(o => o.ScriptPubKey == change.ScriptPubKey);
+				if (request.DonateChangeToMiners && hasChange)
+				{
+					var txTmp = psbt.GetGlobalTransaction();
+					txTmp.Outputs.RemoveAll(o => o.ScriptPubKey == change.ScriptPubKey);
+					psbt = txBuilder.CreatePSBTFrom(txTmp, false);
+					hasChange =  psbt.Outputs.Any(o => o.ScriptPubKey == change.ScriptPubKey);
```
Comment (NicolasDorier) [inline NBXplorer/Controllers/MainController.PSBT.cs:332]: `hasChange = false` no ?

**Comment by NicolasDorier** [comment]
I am curious, why do you need this?

File: NBXplorer/Controllers/MainController.PSBT.cs (hunk 1 above)
Comment (Kukks) [reply NBXplorer/Controllers/MainController.PSBT.cs:332]: You always have to define a change address to the tx builder if there is not an exact spend, so has change is almost always true unless the requests is specifically crafted. I then remove the change output manually

**Comment by Kukks** [comment]
> I am curious, why do you need this?

It is a request from @silentlink

**Pull request closed without merging**